File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>ans(26,0);
        for(auto ch : s) ans[ch-'a']++;
        for(auto ch : t) ans[ch-'a']--;
        for(auto f : ans){
            if(f!=0) return false;
        }
        return true;
    }
};

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] alphabets = new int[26];

        for(char ch : s.toCharArray()) {
            alphabets[ch - 'a']++;
        }

        for(char ch : t.toCharArray()) {
            alphabets[ch-'a']--;
        }

        for(int val : alphabets) {
            if(val!=0) return false;
        }

        return true;
    }
}